src/screens/SignInScreen/index.js

```diff
@@ -12,7 +12,6 @@ class SignInScreen extends Component {
     const { login, navigation, email, password } = this.props
     try {
       await login({ email, password })
```

Comment:
```suggestion
      login({ email, password })
```
It seems this function doesn't need to be async.